SonarRunner.Shim/SonarRunner.Shim.csproj

```diff
@@ -31,6 +31,9 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject />
```

Comment:
The "start-up project" setting is stored in the user's .suo file which isn't checked in. This property was added automatically by VS2015 for something for some other reason (no idea what).